modules/planning/common/path/discretized_path.cc

```diff
@@ -29,9 +29,12 @@ namespace planning {
 
 using apollo::common::PathPoint;
 
-DiscretizedPath::DiscretizedPath(const std::vector<PathPoint> &path_points)
+DiscretizedPath::DiscretizedPath(const std::vector<PathPoint>& path_points)
```

Comment:
You can combine these two constructors by using a single pass by value constructor like this:
DiscretizedPath::DiscretizedPath(std::vector<PathPoint> path_points) : std::vector<PathPoint>(std::move(path_points)) {}

Reply:
@YajiaZhang 
I just follow the format of apollo::map::Path way.
Right. The pass by value way is more concise with little extra one move construction for pass-by-right-value or move assignment for pass-by-left-value.
I will update it to the pass by value way.

Reply:
@YajiaZhang 
Done.